src/test/java/Homework24.java

```diff
@@ -0,0 +1,18 @@
+import org.testng.annotations.Test;
+import pages.BasePage;
+import pages.LoginPage;
+import pages.PlaylistPage;
+
+public class Homework24 extends BaseTest {
+    String playlistName;
+    LoginPage loginPage;
+    BasePage basePage;
+    PlaylistPage playlistPage;
+
+    @Test
+    public void login() {
+        loginPage = new LoginPage(driver);
+        loginPage.loginWithCorrectCred();
+    }
```

Comment:
Please, add Assert

**One test - One Assert**

For example

![Clean_Test](https://github.com/Test-Pro-Bootcamp/A49-QA-Automation-Framework/assets/104761482/b0656807-1874-4bf5-9d73-058830c8e876)